test/setup/index.ts

```diff
@@ -9,6 +9,7 @@ import * as waiters from './waiters';
 
 const testPlatform = process.env['TEST_PLATFORM'] || 'kind';
 const createCluster = process.env['CREATE_CLUSTER'] === 'true';
+const deploymentType = process.env['DEPLOYMENT_TYPE'] || 'YAML';
```

Comment:
I don't want to block on this - but I'd love it if you could push for paying back some technical debt around these areas in the next sprint. about one day's work at most?
1. unify the setup parameters under one object:
```
interface setupConfiguration = {
  platform: string;
  createCluster: bool;
  deploymentType: string;
}
```
or something like that.

2. find a more elegant way to inject these values than having 3 env variables blocking our package.json

what do you think?

Reply:
Sure, let's do it with some YAML config! 🙂 

Agreed, will be happy to allocate some time to resolve this

Reply:
not sure if this still needs to be done or not, either as a ticket or a commit :)